lte/gateway/c/core/oai/tasks/mme_app/mme_app_context.cpp

```diff
@@ -376,23 +376,19 @@ struct ue_mm_context_s* mme_ue_context_exists_s11_teid(
 //------------------------------------------------------------------------------
 ue_mm_context_t* mme_ue_context_exists_guti(
     mme_ue_context_t* const mme_ue_context_p, const guti_t* const guti_p) {
-  hashtable_rc_t h_rc = HASH_TABLE_OK;
-  uint64_t mme_ue_s1ap_id64 = 0;
+  uint32_t mme_ue_s1ap_id = INVALID_MME_UE_S1AP_ID;
 
   char guti_str[GUTI_STRING_LEN] = {0};
   convert_guti_to_string(guti_p, &guti_str);
-  h_rc = obj_hashtable_uint64_ts_get(mme_ue_context_p->guti_ue_context_htbl,
-                                     (const void*)guti_str, strlen(guti_str),
-                                     &mme_ue_s1ap_id64);
 
-  if (HASH_TABLE_OK == h_rc) {
-    return mme_ue_context_exists_mme_ue_s1ap_id(
-        (mme_ue_s1ap_id_t)mme_ue_s1ap_id64);
+  mme_ue_context_p->guti_ue_context_map.get(guti_str, &mme_ue_s1ap_id);
+  if (INVALID_MME_UE_S1AP_ID != mme_ue_s1ap_id) {
+    return mme_ue_context_exists_mme_ue_s1ap_id(mme_ue_s1ap_id);
   } else {
-    OAILOG_WARNING(LOG_MME_APP, " No GUTI hashtable for GUTI ");
+    OAILOG_WARNING(LOG_MME_APP, " Failed to get ue context for guti: " GUTI_FMT,
```

Comment:
Correct the log, here it's failing to get mme_ue_s1ap_id not the pointer to ue context